worlds/witness/player_items.py

```diff
@@ -216,8 +216,8 @@ def get_progressive_item_ids_in_pool(self) -> Dict[int, List[int]]:
             if isinstance(item.definition, ProgressiveItemDefinition):
                 # Note: we need to reference the static table here rather than the player-specific one because the child
                 #   items were removed from the pool when we pruned out all progression items not in the settings.
-                output[cast(int, item.ap_code)] = [cast(int, static_witness_items.ITEM_DATA[child_item].ap_code)
-                                                   for child_item in item.definition.child_item_names]
+                output[cast_not_none(item.ap_code)] = [cast(int, static_witness_items.ITEM_DATA[child_item].ap_code)
```

Comment:
Is this another one of the same thing inside the list comprehension?

Reply:
Lol yes